pkg/kubelet/oom/oom_watcher_linux_test.go

```diff
@@ -57,8 +57,8 @@ func TestWatcherRecordsEventsForOomEvents(t *testing.T) {
 			Pid:                 1000,
 			ProcessName:         "fakeProcess",
 			TimeOfDeath:         time.Now(),
-			ContainerName:       recordEventContainerName,
-			VictimContainerName: "some-container",
+			ContainerName:       recordEventContainerName + "/some-container",
```

Comment:
`ContainerName` will be `//some-container` (i.e. two slases as a prefix). Is that intended?

Reply:
Good catch, I fixed it here and elsewhere.